src/Logging/src/DynamicLogger/DynamicILogger.cs

```diff
@@ -13,47 +13,30 @@
 // limitations under the License.
 
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Console;
-using Microsoft.Extensions.Logging.Console.Internal;
 using System;
 using System.Collections.Generic;
-using System.Runtime.CompilerServices;
 
 namespace Steeltoe.Extensions.Logging
 {
-    public class DynamicConsoleLogger : ILogger
+    public class DynamicILogger : ILogger
```

Comment:
Howabout DynamicLogger instead of DynamicILogger?

Reply:
Just putting it back the way it was